packages/logger/package.json

```diff
@@ -19,8 +24,9 @@
     "url": "https://github.com/nasa/cumulus"
   },
   "scripts": {
-    "build": "true",
-    "test": "ava"
+    "tsc": "tsc",
+    "prepare": "npm run tsc",
+    "test": "npm run tsc && ava"
```

Comment:
I half wonder if we should follow a pattern of test explicitly running prepare.... except that's not great for more complicated packages. 

I suppose running tsc isn't *too* bad on test.... something like Jest to avoid it is probably not what we want to do now.

Reply:
I think it should probably be taken out, so that people can use nodemon or something similar to build their .js files as they make changes to .ts, rather than running tsc before every test run. I'll remove this.